velox/docs/monthly-updates/january-2023.rst

```diff
@@ -0,0 +1,82 @@
+********************
+January 2023 Update
+********************
+
+Core Library
+============
+
+* Support for Generic writer.
+* Reject null aware right semi project joins and add support for null-aware left semi join project with filter.
+* Various memory pool fixes : to always return valid buffer, memory usage count leak with async data cache failures.
```

Comment:
`Fix memory pool to always return a valid buffer, and usage count leak with async data cache failures.`